src/common/src/error.rs

```diff
@@ -70,6 +71,15 @@ impl Display for TrackingIssue {
     }
 }
 
+#[derive(Error, Debug, Macro)]
```

Comment:
`Macro` isn't a very cool name 😕

Reply:
Do you have any proposals? 😢 It's not stable, so feel free to change.